docs/service-bundle.md

```diff
@@ -0,0 +1,225 @@
+# Service Bundle Specification
+
+A Service Bundle is a container image that the broker can use to manage
+deployment of a service in a cluster. It adheres to the properties and
+behaviors described below.
+
+## Label
+
+Every service bundle container image has a label ‘com.redhat.apb.spec’ that
+contains a base64 encoded string. This label holds [important
+metadata](https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/blob/master/docs/developers.md#apb-spec-file)
+related to the service bundle. Example:
+
+```
```

Comment:
Does the `apb.yml` become `bundle.yml`?

Reply:
That would be part of a proposal doc. So as far as I can tell, it should remain apb.yml until we decide.